instrumentation/spring/spring-boot-autoconfigure/README.md

```diff
@@ -390,51 +390,105 @@ If an exporter is present in the classpath during runtime and a spring bean of t
-| Logging Exporter | otel.exporter.logging.enabled               | `true`        | LoggingSpanExporter    |
+##### Enabling/Disabling Exporters
+
+All exporters can be enabled or disabled as in the
+[SDK auto-configuration](https://github.com/open-telemetry/opentelemetry-java/blob/main/sdk-extensions/autoconfigure/README.md#exporters).
+This is the preferred way to enable/disable exporters and takes precedence over the properties below.
+
+| Feature               | Property                                    | Default Value | ConditionalOnClass        |
+|-----------------------|---------------------------------------------|---------------|---------------------------|
+| Otlp Exporter         | otel.exporter.otlp.enabled                  | `true`        | -                         |
+| Otlp Span Exporter    | otel.exporter.otlp.traces.enabled           | `true`        | OtlpGrpcSpanExporter      |
+| Otlp Metrics Exporter | otel.exporter.otlp.metrics.enabled          | `true`        | OtlpGrpcMetricExporter    |
+| Otlp Logs Exporter    | otel.exporter.otlp.logs.enabled             | `true`        | OtlpGrpcLogRecordExporter |
+| Jaeger Exporter       | otel.exporter.jaeger.enabled                | `true`        | JaegerGrpcSpanExporter    |
+| Zipkin Exporter       | otel.exporter.zipkin.enabled                | `true`        | ZipkinSpanExporter        |
+| Logging Exporter      | otel.exporter.logging.enabled               | `false`       | LoggingSpanExporter       |
 
 <!-- Slf4j Log Correlation  otel.springboot.loggers.slf4j.enabled		true   		org.slf4j.MDC -->
 
-##### Resource Properties
+##### Enabling/Disabling Features
 
-| Feature  | Property                                         | Default Value          |
-| -------- | ------------------------------------------------ | ---------------------- |
-| Resource | otel.springboot.resource.enabled                 | `true`                 |
-|          | otel.springboot.resource.attributes.service.name | `unknown_service:java` |
-|          | otel.springboot.resource.attributes              | `empty map`            |
+| Feature               | Property                                    | Default Value | ConditionalOnClass        |
+|-----------------------|---------------------------------------------|---------------|---------------------------|
+| spring-web            | otel.instrumentation.spring-webmvc.enabled  | `true`        | RestTemplate              |
+| spring-webmvc         | otel.instrumentation.spring-web.enabled     | `true`        | OncePerRequestFilter      |
+| spring-webflux        | otel.instrumentation.spring-webflux.enabled | `true`        | WebClient                 |
+| @WithSpan             | otel.instrumentation.annotations.enabled    | `true`        | WithSpan, Aspect          |
 
-`unknown_service:java` will be used as the service-name if no value has been specified to the
```

Comment:
It would be great to keep this information to help users diagnose this issue

Reply:
it's here: https://github.com/zeitlinger/opentelemetry-java-instrumentation/blob/95e8c0faed45fbe35238c9e7ef9eb7b9fa81bf36/instrumentation/spring/spring-boot-autoconfigure/README.md?plain=1#L461